exchange/bidder.go

```diff
@@ -135,6 +135,13 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb2.B
 		return nil, errs
 	}
 
+	// #1712 add Sec-GPC to the headers if set in original request
+	if reqInfo.SecGPCFlag == "1" {
+		for i := 0; i < len(reqData); i++ {
+			reqData[i].Headers.Add("Sec-GPC", reqInfo.SecGPCFlag)
+		}
+	}
```

Comment:
Passing the header to adapters isn't mention in the PBS spec. I personally have no objection if this is useful. Please first propose the change in the https://github.com/prebid/prebid-server/issues/1712 thread.

Reply:
Could you please propose this feature in https://github.com/prebid/prebid-server/issues/1712? This behavior is not currently in the spec.

Reply:
I apologize. This is the original proposal. I had misread the original intent. We're all good on this item.